### ISSUE TITLE
New Rule: newline-above-import

### ISSUE THREAD

**Issue by Deadmano** [import-js/eslint-plugin-import#3137]
Any chance for a rule that allows for adding a newline above the first import?

**Comment by ljharb** [comment]
For what purpose?

**Comment by Deadmano** [comment]
@ljharb I end up having a JSDoc comment at the top of each file that has a license:
```JS
/**
 * @license MIT
 * This file is part of Example Project.
 * Copyright © Author
 */
import { CommentState, FormattingState, LineState } from '../config/types';
```
I'd like to ensure that if there is no space above the first import, one gets added (for consistency). The same way rules such as `newline-after-import` handle it, but in reverse.

**Comment by ljharb** [comment]
The  “after” rule is because command line tools work better when all files have a trailing newline, not for aesthetics or anything.

Since per-file license boilerplate is a bad practice, I’m not sure that’s sufficiently motivating reason to add an entirely new rule.
